README.md

```diff
@@ -66,6 +66,21 @@ $ foxglove devices list
     | dev_lwjzOMxryMmP3yXg | nuScenes-v1.0-mini | 2021-12-09T21:45:51Z | 2021-12-09T21:45:51Z |
 ```
 
+### Projects
+
+Projects provide data isolation and access control to resources. See if your organization
+has [Projects](https://docs.foxglove.dev/docs/projects) enabled.
```

Comment:
This is a very minor nit but this phrasing makes me think that I can check a box and enable projects for my organization (as opposed to potentially needing to upgrade to a new plan).

What about "See if your organization supports projects" or "see if your organization's plan supports projects"?

Reply:
Yeah this sounds better, thank you!